CONTRIBUTING.md

```diff
@@ -1,41 +1,77 @@
 # Contributing to Okta Open Source Repos
 
-## Style
+Thank you for your interest in contributing to Okta's Open Source Projects! Before submitting a PR, please take a moment to read over our [Contributer License Agreement](https://developer.okta.com/cla/). In certain cases, we ask that you [sign a CLA](https://developer.okta.com/sites/all/themes/developer/pdf/okta_individual_contributor_license_agreement_2016-11.pdf) before we merge your pull request.
+
+- [Commit Message Guidelines](#commit-message-guidelines)
```

Comment:
FYI there is a VSCode extension called "Markdown all in one" that can autogenerate these TOC's whenever the document changes. I am using it on the SIW side.